new/LIT6822SalamLakaAqabi.xml

```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6822SalamLakaAqabi" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Salām la-ka ʿaqābi</title>
```

Comment:
laka 
could you update the title to have fidal and transliteration?

Reply:
I deleted the hyphen, so now it is "laka" (not "la-ka"). 
How can I give the title in fidal in addition to the title in transliteration? Most of the Salam, that I checked have an English title without transliteration or fidal title.

Reply:
```
  <title xml:lang="gez" xml:id="t1">ሰላም፡ ለከ፡ ዐቃቢ፡</title>
  <title xml:lang="gez" corresp="#t1">Salām laka ʿaqābi...</title>
```

(Adding ... to titles that are simply the first words of the text in question isn't prescribed anywhere, but Denis taught it to me for the DS Catalogue, and I find it very reasonable)

Reply:
Me too, I find it very reasonable and did it in the way, you recommended!